test/cfg/windows.cpp

```diff
@@ -146,6 +146,11 @@ void validCode()
     SecureZeroMemory(byteBuf, sizeof(byteBuf));
     RtlFillMemory(byteBuf, sizeof(byteBuf), 0xff);
 
+    // Intrinsics
+    __noop();
+    __noop(1, "test", NULL);
```

Comment:
__noop with 3 parameters is not handled by the configuration is it?

Reply:
It is handled by the configuration. If no arg entry exists it is handled like an empty arg entry with nr=any set. At least this is what i observed. Otherwise many configurations for example in qt.cfg and gtk.cfg would not make much sense and wouldn't work. Is this not the intended behavior? 
I already wondered how to specify that a function configuration is meant for a function without parameters.

Reply:
This is the command line for the first windows.cfg test:
~~~
./cppcheck --check-library --enable=information --enable=style --error-exitcode=1 --suppress=missingIncludeSystem --inline-suppr '--template="{file}:{line}:{severity}:{id}:{message}"' --inconclusive --platform=win32A test/cfg/windows.cpp
~~~
Since `--check-library` and `--enable=information` is enabled i would expect an information message about a missing function configuration for `__noop(1, "test", NULL);` if the configuration would not handle this case.

Reply:
ok.

I do not want to break all existing configurations but I think it's unfortunate that you can have 3 arguments in the code and no `<arg>`. I would like that we change this.. however carefully.

Reply:
so for now I have no complaints about this PR. Feel free to apply it.

Reply:
I see and fully agree, i will add `<arg nr="any"/>` to the __noop configuration before merging.
I could also create a ticket that this behaviour should be carefully changed.